### PR TITLE
add spring gem

### PR DESCRIPTION
**Pull request by tamata78**
factory girlの設定を活かすためにspring gemの設定を行った。

### CODE REVIEW

File: bin/rake [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env ruby
+begin
+  load File.expand_path("../spring", __FILE__)
```
Comment (freezer530000) [inline bin/rake]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/rake [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+rescue LoadError
```
Comment (freezer530000) [inline bin/rake:4]: **[rubocop]**
- Do not suppress exceptions.

File: bin/rails [hunk 3]
```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env ruby
+begin
+  load File.expand_path("../spring", __FILE__)
```
Comment (freezer530000) [inline bin/rails]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/rails [hunk 4]
```diff
@@ -1,4 +1,8 @@
...
+rescue LoadError
```
Comment (freezer530000) [inline bin/rails:4]: **[rubocop]**
- Do not suppress exceptions.

File: bin/rspec [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+begin
+  load File.expand_path("../spring", __FILE__)
```
Comment (freezer530000) [inline bin/rspec]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/rspec [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+rescue LoadError
```
Comment (freezer530000) [inline bin/rspec:4]: **[rubocop]**
- Do not suppress exceptions.

File: bin/spring [hunk 7]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
+  require "rubygems"
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/spring [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  require "bundler"
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/spring [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+  if match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Assignment in condition - you probably meant to use `==`.
- Line is too long. [95/80]

File: bin/spring [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+    Gem.paths = { "GEM_PATH" => [Bundler.bundle_path.to_s, *Gem.path].uniq }
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/spring [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+    gem "spring", match[1]
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/spring [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+    require "spring/binstub"
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: bin/rails [hunk 13]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env ruby
+begin
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError
+    puts 'load error'
```
Comment (freezer530000) [inline bin/rails]: **[rubocop]**
- Use 2 (not 4) spaces for indentation.

File: bin/spring [hunk 14]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
+  require 'rubygems'
+  require 'bundler'
+
+  if match = Bundler.default_lockfile.read
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Assignment in condition - you probably meant to use `==`.

File: bin/spring [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+    .match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
```
Comment (freezer530000) [inline bin/spring]: **[rubocop]**
- Align the operands of a condition in an `if` statement spanning multiple lines.

**Comment by bakunyo** [comment]
springの起動確認できました〜 :+1: 

LGTM

**Comment by t-tonchim** [comment]
同じく確認できたのでマージします！

**Pull request merged**